Sources/HAPUpdate/Inspector.swift

```diff
@@ -366,7 +366,22 @@ public class Inspector {
         for category in categoryInfo.sorted(by: { $0.id < $1.id }) {
             write("\tcase \(categoryName(category.name)) = \"\(category.id)\"")
         }
-        write("}")
+        write("""
+        }
+
+        extension AccessoryType: CustomStringConvertible {
+            public var description: String {
+                switch self {
+        """)
+        for category in categoryInfo.sorted(by: { $0.id < $1.id }) {
+            write("\t\tcase .\(categoryName(category.name)):")
```

Comment:
We use 4 spaces for indentation, not a tab.

Reply:
What ? The Inspector has written tab's since its inception, there are 168 occurrences of '\t' in the class....

Reply:
I see: the file _is_ using tabs. However this PR introduces mixed-style indentation.

This is also a major reason I use space indents everywhere. While I'm fine with this PR using tabs (throughout), I'll probably update the code to replace tabs by spaces to align with the overall code in this project.

Reply:
OK I've replaced all the \t with four spaces